ballerina/main.bal

```diff
@@ -31,7 +31,7 @@ type DefaultOpenAIModelConfig record {|
     string model;
 |};
 
-public annotation map<json> Schema on type;
+public annotation map<json> JsonSchema on type;
```

Comment:
If we name this as `JsonSchema` we needs to change the documentation of the `expectedResponseTypedesc` parameter as JSON schema

Reply:
Why? That is meant to work with anydata.